.github/workflows/integrational-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Integrational tests
+on:
+  push:
+    paths-ignore:
+      - 'doc/**'
+
+jobs:
+  run-tests:
+    name: Run Intergrational tests using Kind
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.7.0"
+
+      - name: Deploy to Kuberenetes
+        run: |-
+          kubectl create configmap hazelcast-configuration --from-file=server/hazelcast.yaml
+          kubectl create secret generic keystore --from-file=server/keystore --from-file=server/truststore --from-literal keystorePassword=123456 --from-literal truststorePassword=123456
+          kubectl create secret generic hz-license-key --from-literal license=${{ secrets.LICENSE }}
+          kubectl apply -f server/rbac.yaml
+          kubectl apply -f server/statefulset.yaml
+
+      - name: Test for successful Deployment
+        run: |-
+          kubectl wait --for=condition=ready pod/hazelcast-0 --timeout=150s
+          kubectl wait --for=condition=ready pod/hazelcast-1 --timeout=150s
```

Comment:
Is that even possible to have these two pods as separate clusters? I mean, once `hazelcast-1` is ready, is it certain that they formed a cluster with `hazelcast-0`?

Reply:
Added the step to deploy the client application and check that it is connected to the cluster with 3 members